addons/escoria-core/game/main_scene.gd

```diff
@@ -5,10 +5,10 @@ extends Node
 
 # Start the main menu
 func _ready():
-	var main_menu_path = ProjectSettings.get_setting(
-		"escoria/ui/main_menu_scene"
-	)
-	var main_menu_scene = load(main_menu_path).instance()
-	escoria.call_deferred("add_child", main_menu_scene)
-	escoria.main_menu_instance = main_menu_scene
+	if escoria.main_menu_instance == null:
```

Comment:
Why is this not in escoria#_ready?

Reply:
Because the main menu needs to be instanced AFTER settings have been loaded so that the language is correct.